amf-cli/js/build-scripts/create-bundle.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
js-build.js file can also be removed as it will only be calling ```sbt cliJS/fullOptJS```. If removed we should adjust release process documentation.

Reply:
I left it because I feel it is valuable to have some place where to do extra things besides the fullOptJS. It value although we start with just one command. Besides it standardizes build and publish process.